extensions/attention-stream/package.json

```diff
@@ -12,13 +14,13 @@
     "build:glean:metrics": "glean translate ./metrics.yaml ./pings.yaml -f typescript -o src/generated",
     "build:glean:docs": "glean translate ./metrics.yaml ./pings.yaml -f markdown -o docs",
     "lint:glean": "glean glinter ./metrics.yaml ./pings.yaml",
-    "build:developer": "cp manifest.dev.json manifest.json && rollup -c --config-enable-developer-mode && npm run tailwind",
```

Comment:
Yep that's intentional, it's going to need a little more work to make it really useful, but I don't think we need manifest changes anymore.

Reply:
The distinction in title was because they were separate extensions so it helped to tell them apart; but if we're merging to one then that bit doesn't matter.